src/Mvc/Mvc.RazorPages/src/ApplicationModels/IPageApplicationModelPartsProvider.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
```

Comment:
Could you add this license header to the file:

```C#
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Reply:
Sure, I'll add it asap.